main.tf

```
@@ -99,7 +100,7 @@ resource "google_compute_backend_service" "default" {
   connection_draining_timeout_sec = lookup(each.value, "connection_draining_timeout_sec", null)
   enable_cdn                      = lookup(each.value, "enable_cdn", false)
   security_policy                 = var.security_policy
-  health_checks                   = [google_compute_health_check.default[each.key].self_link]
+  health_checks                   = lookup(each.value, "health_check", null) == null ? [] : [google_compute_health_check.default[each.key].self_link]
```

Can we simplify this to
```suggestion
  health_checks                   = lookup(each.value, "health_check", false) ? [] : [google_compute_health_check.default[each.key].self_link]
```

---

Done, @bharathkkb just to confirm, looking up a null value will return the default (false in this case)?